packages/kit/src/runtime/client/parse.js

```diff
@@ -2,25 +2,30 @@ import { exec, parse_route_id } from '../../utils/routing.js';
 
 /**
  * @param {import('types').CSRPageNodeLoader[]} nodes
- * @param {Record<string, [number[], number[], number, 1?]>} dictionary
+ * @param {Record<string, [number, number[]?, number[]?, 1?]>} dictionary
```

Comment:
From above, we could then reuse it here (looks a bit awkward) and it would give more informative hover types / autocompletion. The `1?` at the end would be removed, though it doesn't seem to be used anymore?

```suggestion
 * @param {typeof import('__GENERATED__/client-manifest.js').dictionary} dictionary
```

Reply:
That's a good idea, it connects the places where it's used, which is always great for navigating the code base. The 1 is obsolete, so another plus for connecting them.